rdbms/src/main/java/io/spine/server/storage/jdbc/StandardMappings.java

```diff
@@ -0,0 +1,136 @@
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.storage.jdbc;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static io.spine.server.storage.jdbc.BaseMapping.baseBuilder;
+import static io.spine.server.storage.jdbc.StandardMappings.DatabaseProductName.MySQL;
+import static io.spine.server.storage.jdbc.StandardMappings.DatabaseProductName.PostgreSQL;
+import static io.spine.server.storage.jdbc.Type.BYTE_ARRAY;
+
+/**
+ * Standard {@linkplain TypeMapping type mappings} for different databases.
+ *
+ * @author Dmytro Grankin
+ */
+public final class StandardMappings {
+
+    private static final SelectableMapping MYSQL_5 = new SelectableMapping(MySQL, 5,
+                                                                           baseBuilder().build());
+    private static final SelectableMapping POSTGRESQL_10 =
+            new SelectableMapping(PostgreSQL, 10,
+                                  baseBuilder().add(BYTE_ARRAY, "BYTEA")
+                                               .build());
+
+    private StandardMappings() {
+        // Prevent instantiation of this utility class.
+    }
+
+    /**
+     * Obtains the type mapping for MySQL 5 database.
+     */
+    @VisibleForTesting
+    public static TypeMapping mySql() {
+        return MYSQL_5;
+    }
+
+    /**
+     * Obtains the type mapping for PostreSQL 10 database.
+     */
+    @VisibleForTesting
+    public static TypeMapping postgreSql() {
+        return POSTGRESQL_10;
+    }
+
+    /**
+     * Obtains the {@linkplain SelectableMapping#suitableFor(DataSourceWrapper) suitable}
+     * type mapping for the specified data source.
+     *
+     * @param dataSource the data source to test suitability
+     * @return the type mapping for the used database
+     *         or MySQL-specific mapping if there is no standard mapping for the database
+     */
+    static TypeMapping get(DataSourceWrapper dataSource) {
+        if (POSTGRESQL_10.suitableFor(dataSource)) {
+            return POSTGRESQL_10;
+        }
+
+        return MYSQL_5;
+    }
+
+    /**
+     * Names of {@linkplain DatabaseMetaData#getDatabaseProductName() database products}.
+     */
+    enum DatabaseProductName {
+
+        PostgreSQL,
+        MySQL
+    }
+
+    /**
+     * A mapping for a {@linkplain #suitableFor(DataSourceWrapper) specific} data source.
+     */
+    static class SelectableMapping implements TypeMapping {
```

Comment:
It has no `select` method. Therefore I don't understand this name.

Reply:
Done.